src/SWP/Bundle/ContentBundle/Resources/config/services.yml

```diff
@@ -4,17 +4,30 @@ parameters:
     swp.service.article.class: SWP\Bundle\ContentBundle\Service\ArticleService
     swp_content_bundle.listener.http_cache_header.class: SWP\Bundle\ContentBundle\EventListener\HttpCacheHeaderListener
     swp_content_bundle.manager.media.class: SWP\Bundle\ContentBundle\Manager\MediaManager
+    swp_content_bundle.router.media.class: SWP\Bundle\ContentBundle\Routing\MediaRouter
```

Comment:
I will leave it like this for now, Later in one PR we should remove all those parameters.